src/components/BasicMapComponent/index.tsx

```diff
@@ -79,7 +86,7 @@ export const BasicMapComponent: React.FC<Partial<MapComponentProps>> = ({
     } catch (error) {
       Logger.error(error);
     }
-  };
+  }, [client, map, t]);
```

Comment:
Is the translation function added here in case the translation - and thus the name of the layer group - changes?

Reply:
yes, this is so the hook is called again if for example the translation function is not available in the first render

see https://reactjs.org/docs/hooks-rules.html